Scoop/Controller/Ride Posting/PostRidePageViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  PostRidePageViewController.swift
+//  Scoop
+//
+//  Created by Tiffany Pan on 3/18/23.
+//
+
+import UIKit
+
+class PostRidePageViewController: UIPageViewController {
+    
+    private var pages = [UIViewController]()
+    
+    override func viewDidLoad() {
+        view.backgroundColor = .white
+        delegate = self
+        dataSource = self
+        
+        let appearance = UIPageControl.appearance()
+        appearance.pageIndicatorTintColor = .systemGray5
+        appearance.currentPageIndicatorTintColor = .systemGray3
+        
+        setupPages()
+        disableSwiping()
+    }
+    
+    private func setupPages() {
+        
+        pages = [
+            InitialPostRideViewController()
+        ]
+        
+        for i in 0..<pages.count {
+            if let page = pages[i] as? OnboardingViewController {
+                page.delegate = self
+            }
+            
+            pages[i] = UINavigationController(rootViewController: pages[i])
+        }
+    }
+    
+    private func disableSwiping() {
+        for view in view.subviews {
+            if let subView = view as? UIScrollView {
+                subView.isScrollEnabled = false
+            }
+        }
+    }
+    
+    private func presentNextVC(_ viewController: UIViewController) {
+        if let viewControllerIndex = pages.firstIndex(of: viewController) {
+            if viewControllerIndex < pages.count - 1 {
+                let vc = pages[viewControllerIndex + 1]
+                self.setViewControllers([vc], direction: .forward, animated: true, completion: nil)
+            }
+        }
+    }
+    
+    private func presentPreviousVC(_ viewController: UIViewController) {
+        guard let viewControllerIndex = pages.firstIndex(of: viewController) else {
+            return
+        }
+            if viewControllerIndex > 0 {
+                let vc = pages[viewControllerIndex - 1]
+                self.setViewControllers([vc], direction: .reverse, animated: true, completion: nil)
+            }
+        }
+    
+}
+
+extension PostRidePageViewController: UIPageViewControllerDelegate {
+    
```

Comment:
Is there a reason why this is here? Maybe add a TODO if there is a potential implementation